app/src/ui/business-card/BusinessCard.tsx

```diff
@@ -0,0 +1,79 @@
+import clsx from "clsx";
+import styles from "./BusinessCard.module.scss";
+import Link from "next/link";
+
+import { Icon } from "ui/icon/Icon";
+import { Card } from "ui/card/Card";
+import { BusinessCardProps } from "./BusinessCard.types";
+import { Typography } from "ui/typography/Typography";
+
+export const BusinessCard: React.FC<BusinessCardProps> = ({ businessLink, content, isExpandable, className }) => {
+  const { businessBackground, businessAvatar, title, location, description, raised, investors, daysLeft, payback } =
+    content;
```

Comment:
I like this pattern, but I prefer: `content.prop` syntax, it is more explicit when used in the actual component prop attribute: `<Typography.Headline1>{content.title}`...